sbin/common/config_init.sh

```diff
@@ -366,6 +367,31 @@ function configDefaults() {
   # The OS kernel name, e.g. 'darwin' for Mac OS X
   BUILD_CONFIG[OS_KERNEL_NAME]=$(uname | awk '{print tolower($0)}')
 
+  # Determine OS full system version
+  local unameSys=$(uname -s)
+  local unameOSSysVer=$(uname -sr)
+  if [ "${unameSys}" == "Linux" ]; then
+    # Linux distribs add more useful distrib /etc/*-release file content
+    set +e
+    if cat /etc/*-release > /dev/null; then
+      local linuxName=""
+      if grep PRETTY_NAME /etc/*-release; then
+        linuxName=$(grep PRETTY_NAME /etc/*-release | cut -d= -f2 | tr -d '"')
+      else
+        linuxName=$(cat /etc/*-release | tr -d '"')
```

Comment:
I assume this isn't going to cause any problems if it's returning multi-line output?

Reply:
mmm possibly, will check

Reply:
Where does it return just `Red Hat`? Not on my RHEL7 and 8 systems ...

Reply:
@sxa here: test-ibmcloud-rhel7-x64-1
```
NAME="Red Hat Enterprise Linux Server"
VERSION="7.9 (Maipo)"
ID="rhel"
ID_LIKE="fedora"
VARIANT="Server"
VARIANT_ID="server"
VERSION_ID="7.9"
PRETTY_NAME="Red Hat"
ANSI_COLOR="0;31"
CPE_NAME="cpe:/o:redhat:enterprise_linux:7.9:GA:server"
HOME_URL="https://www.redhat.com/"
BUG_REPORT_URL="https://bugzilla.redhat.com/"

REDHAT_BUGZILLA_PRODUCT="Red Hat Enterprise Linux 7"
REDHAT_BUGZILLA_PRODUCT_VERSION=7.9
REDHAT_SUPPORT_PRODUCT="Red Hat Enterprise Linux"
REDHAT_SUPPORT_PRODUCT_VERSION="7.9"
```